.github/workflows/test.yaml

```diff
@@ -290,7 +290,30 @@ jobs:
         name: ${{ matrix.python-version || env.DEFAULT_PYTHON_VERSION }}-${{matrix.job-name || 'default-tests' }}
         fail_ci_if_error: true
 
+  benchmarks:
```

Comment:
Note that with the current configuration, if the benchmark tests are failing, the merge is not blocked.

I think that we should add the benchmark tests to  `all-successful` list

```
  all-successful:
    # Is very important to force running this always, as otherwise it will be
    #  skipped by default.
    if: always()
    runs-on: ubuntu-latest
    # Here should be the list of all the other jobs defined in this file.
    needs:
      - testing
      - narrativedocs
      - apidocs
      - static-checks
      - release-publish
```